javascript/sdk/lib/types.ts

```diff
@@ -211,6 +211,16 @@ export interface TestData {
   mappings: string[];
 }
 
+export interface TestUsage {
+  count: number;
+  failed: number;
+}
+
+export interface TestActivity {
+  id: string;
+  usafe: TestUsage;
```

Comment:
usage

Reply:
wow and thats why we review :P Thanks Mahina!